helpers/lagoonenvclean.sh

```diff
@@ -0,0 +1,182 @@
+#!/bin/bash
+#
+# Author: Vincenzo De Naro Papa
+# Email: vincenzo.denaropapa@amazee.io
+#
+# Description: Run a Lagoon old environments cleanup based on closed GitHub's PRs
+# Usage: ./lagoonenvclean customer number_of_months yes|no 
+#
+# Example: ./lagoonenvclean rasch 3 yes 
+# Above command, will revome all environments, older than 3 months, in all customer's projects, related to closed PRs on GitHub.
+
+
+# Script accepts 3 arguments (1 mandatory):
+#
+# CUSTOMER to query
+# MONTHS to calculate the time slot
+# DRYRUN to execute or not the deletion of the environments
+
+
+CUSTOMER="$1"
+MONTHS="${2:-2}"
+DRYRUN="${3:-false}"
+
+
+# CLEANDATE: environments older than that date will be remove if related to a closed PR
+# ENDPOINT: Lagoon API endpoint to query
+# GHENDPOINT: GitHub API endpoint
+# AMZBEARERTKN: Lagoon API token
+# GHUTKN: GitHub API Authentication token
+
+CLEANDATE=$(date +%Y%m%d -d "-$MONTHS months")
+ENDPOINT="https://api-lagoon-master.lagoon.ch.amazee.io/graphql"
+GHENDPOINT="https://api.github.com"
+AMZTKN=""
+AMZBEARERTKN="Authorization: bearer $AMZTKN"
+GHTKN=""
+GHBEARERTKN="Authorization: bearer $GHTKN"
```

Comment:
I would expect some kind of explanation what type of Github Token is actually needed (which permissions does the token need)

Reply:
I added in the body of the script what kind of permissions are needed.